extensions/wikia/AdEngine/js/video/vastParser.js

```diff
@@ -6,18 +6,16 @@ define('ext.wikia.adEngine.video.vastParser', [
 	'use strict';
 	var logGroup = 'ext.wikia.adEngine.video.vastParser';
 
-	function getFirstNumber(possibleValues) {
+	function getLastNumber(possibleValues) {
```

Comment:
How about the scenario when `possibleValues` will have only not a number elements? Previously we've returned an empty string now it can be an `undefined`, I think.

Reply:
Great catch 💎